geode-management/src/main/java/org/apache/geode/management/configuration/Region.java

```diff
@@ -89,6 +119,14 @@ public void setType(RegionType type) {
     this.type = type;
   }
 
+  public int getRedundantCopies() {
```

Comment:
You have javadocs on the private field but then no javadocs on the external public methods